### DISCUSSION TITLE
Need a docker image version for 0.11.1

### DISCUSSION THREAD

**Discussion by wineternity** [milvus-io/milvus#6029] [Q&A and General discussion]
Hello, we are current use 0.11.0 which is no longer supported, but we need some bug fix like getEntityById is almost 2S which is too low ( https://github.com/milvus-io/milvus/pull/4489  )
I could not find the branch for 0.11.1

Could you provided a docker version for 0.11.1 or give me the commit to build the docker image from the source code?

And is there any instruction for build docker image?

**Comment by yhmo** [comment]
I remember the 0.11.1 was renamed to "experimental" branch, not very sure. You can build docker image by this file:
https://github.com/milvus-io/milvus/blob/experimental/docker/deploy/cpu/ubuntu18.04/Dockerfile

Also can choose other OS such as CentOS and ubuintu16.

**Comment by yhmo** [comment]
Enter the path `/milvus/docker/deploy/cpu/ubuntu18.04` from the terminal, run the command to build an image:
`docker build -t milvusdb/milvus:0.11.1 .`

**Reply by wineternity** [reply]
thanks so much, I will check this branch to see whether bug is fixed and build the image

**Reply by wineternity** [reply]
Another question , If I use 0.11.1, does python or java sdk need to change?  How can I get the sdk?

**Reply by yhmo** [reply]
The pymilvus old version for 0.11.x has been removed. Only the java sdk is available.
https://github.com/milvus-io/milvus-sdk-java/tree/v0.9.1
The java sdk v0.9.1 is for 0.11.0, I think it also can work with 0.11.1.

But I strongly recommend you wait for the upcoming milvus v2.0. The release date of milvus v2.0 RC version is 2021-06-26.
Although the v2.0 RC(release candidate version) is not a official release, it may contains some unstable issues, but more developers will put efforts to fix issues, enhance it. It is much better than the 0.11.x without maintaince. 
And with more and more features implemented, the v2.0 will be more powerful than the 0.11.x, you can check the features of v2.0 here: https://github.com/milvus-io/milvus#key-features

**Reply by wineternity** [reply]
I will upgrade to v2.0 RC as soon as I can,  but the current problem must be fixed right now. So We should make it work first.
